server/app.js

```diff
@@ -1,13 +1,14 @@
 const express = require('express');
 const app = express();
+// const path = require('path');
 
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 app.use(express.static(__dirname + '/../public'));
-// app.use(express.static(__dirname + '/../client/dist'));
+// app.use(express.static(path.join(__dirname, 'public')));
 
```

Comment:
This is great.  This is exactly what I was referring to in an earlier comment.  You can disregard that comment.